js/views/modals/About/Donations.js

```diff
@@ -16,34 +18,27 @@ export default class extends baseVw {
 
   events() {
     return {
-      'click #copyAddress': 'copyDonationAddress',
-      'click #donationAddress': 'copyDonationAddress',
+      'click .js-copyAddress': 'copyDonationAddress',
     };
   }
 
   copyDonationAddress() {
-    const address = document.querySelector('#donationAddress');
-    const range = document.createRange();
-    const selection = window.getSelection();
-    range.selectNodeContents(address);
-    selection.removeAllRanges();
-    selection.addRange(range);
-
     clipboard.writeText(obDonationAddress);
 
-    this.$('#copyNotification').addClass('active');
+    this.$('.copyNotification').addClass('active');
```

Comment:
Okay.
@rmisio thanks for the very useful explanation. You are always so generous with your time. I really appreciate that.